### PR TITLE
Create self-hosting.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zhangx501** [comment]
/assign @chenrui333
Please give approval to this PR.

File: content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+configured in the kubelet via static files.
+-->
+kubeadm 允许您实验性地创建 _self-hosted_ Kubernetes 控制平面。
+这意味着 API 服务器，控制管理器和调度程序之类的关键组件将通过配置 Kubernetes API 以 [DaemonSet pods](/docs/concepts/workloads/controllers/daemonset/) 的身份运行，而不是通过静态文件将 [static pods](/docs/tasks/administer-cluster/static-pod/) 在 kubelet 中配置。
+
+<!--
+To create a self-hosted cluster see the
+[kubeadm alpha selfhosting pivot](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-selfhosting) command.
+-->
+要创建自托管集群，请参见 [kubeadm alpha 自托管枢纽](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-selfhosting) 命令。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md]: `selfhosting pivot` is part of the command, should not be translated.

File: content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+<!--
+The self-hosting bootstrap process is documented in the [kubeadm design
+document](https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.9.md#optional-self-hosting).
+-->
+自托管引导过程记录在 [kubeadm 设计文档](https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.9.md#optional-self-hosting) 中。
+
+<!--
+In summary, `kubeadm alpha selfhosting` works as follows:
+-->
+总而言之，`kubeadm alpha 自托管` 的工作原理如下：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md]: `selfhosting` should not be translated

File: content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  1. 等待此引导静态控制平面运行且良好。
+     这与没有自我托管的 `kubeadm init` 过程相同。
+<!--
+  1. Uses the static control plane Pod manifests to construct a set of
+    DaemonSet manifests that will run the self-hosted control plane.
+    It also modifies these manifests where necessary, for example adding new volumes
+    for secrets.
+-->
+  1. 使用静态控制平面 Pod 清单来构造一组 DaemonSet 清单，这些清单将运行自托管的控制平面。
+     它还会在必要时修改这些清单，例如添加新的秘密卷。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/self-hosting.md]: retranslate 'volumes for secrets' ...

**Pull request merged**